incubator-tools/schema_comparision/schema_comparision.ipynb

```diff
@@ -0,0 +1,346 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.       Base_schema={"displayName": "invoice_uptrain", "entityTypes": [{"type": "amount_due", "baseType": "money", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "amount_paid_since_last_invoice", "baseType": "money", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "carrier", "baseType": "string", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "currency", "baseType": "currency", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "currency_exchange_rate", "baseType": "money", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "customer_tax_id", "baseType": "string", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "receiver_tax_id", "baseType": "string", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "delivery_date", "baseType": "datetime", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "due_date", "baseType": "datetime", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "freight_amount", "baseType": "money", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "invoice_date", "baseType": "datetime", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "invoice_id", "baseType": "string", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "net_amount", "baseType": "money", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "payment_terms", "baseType": "string", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "purchase_order", "baseType": "string", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "receiver_address", "baseType": "address", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "receiver_email", "baseType": "string", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "receiver_name", "baseType": "string", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "receiver_phone", "baseType": "string", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "receiver_website", "baseType": "string", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "remit_to_address", "baseType": "address", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "remit_to_name", "baseType": "string", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "ship_from_address", "baseType": "address", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "ship_from_name", "baseType": "string", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "ship_to_address", "baseType": "address", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "ship_to_name", "baseType": "string", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "supplier_address", "baseType": "address", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "supplier_email", "baseType": "string", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "supplier_iban", "baseType": "string", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "supplier_name", "baseType": "string", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "supplier_payment_ref", "baseType": "string", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "supplier_phone", "baseType": "string", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "supplier_registration", "baseType": "string", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "supplier_tax_id", "baseType": "string", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "supplier_website", "baseType": "string", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "total_amount", "baseType": "money", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "total_tax_amount", "baseType": "money", "occurrenceType": "OPTIONAL_ONCE"}, {"type": "line_item/amount", "baseType": "money", "occurrenceType": "OPTIONAL_MULTIPLE"}, {"type": "line_item/description", "baseType": "string", "occurrenceType": "OPTIONAL_MULTIPLE"}, {"type": "line_item/product_code", "baseType": "string", "occurrenceType": "OPTIONAL_MULTIPLE"}, {"type": "line_item/purchase_order", "baseType": "string", "occurrenceType": "OPTIONAL_MULTIPLE"}, {"type": "line_item/quantity", "baseType": "number", "occurrenceType": "OPTIONAL_MULTIPLE"}, {"type": "line_item/unit", "baseType": "string", "occurrenceType": "OPTIONAL_MULTIPLE"}, {"type": "line_item/unit_price", "baseType": "money", "occurrenceType": "OPTIONAL_MULTIPLE"}, {"type": "vat/amount", "baseType": "money", "occurrenceType": "OPTIONAL_MULTIPLE"}, {"type": "vat/category_code", "baseType": "string", "occurrenceType": "OPTIONAL_MULTIPLE"}, {"type": "vat/tax_amount", "baseType": "money", "occurrenceType": "OPTIONAL_MULTIPLE"}, {"type": "vat/tax_rate", "baseType": "money", "occurrenceType": "OPTIONAL_MULTIPLE"}]}
</pre><p>These schemas could be moved into a JSON file to make this easier to read.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/682/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>